packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -185,7 +191,7 @@ class Auth {
         // Special endpoint for injecting SLAS private client secret.
         const baseUrl = config.proxy.split(MOBIFY_PATH)[0]
         const privateClientEndpoint = `${baseUrl}${SLAS_PRIVATE_PROXY_PATH}`
-
+        const keysExcludedFromSuffixing = ['dw_dnt']
```

Comment:
Can you please add a comment explains why we do this? It is better to keep a constant for the key val. Image if we want to change the key name in the future, we have to do it several places, if we miss one place, a bug happens

Reply:
@alexvuong can you point out the multiple lines where you would need to make the change? It looks to me like the code **does** use a `const` so it should make it possible to change the value everywhere by just changing it here.

Also, it's not very changeable, since SLAS defines the key name in the JWT claims

Reply:
I thought @alexvuong was referring to `dw_dnt` alone being set as a const in Auth class. In another comment, she asked to put all logic in `updateDNT` into Auth.setDnt, so within the class all references to `dw_dnt` would refer to it by a const. 

The JWT claim is `dnt` and not `dw_dnt`, but it's still not very changeable because ECOM codebase has the cookie name as `dw_dnt` as well.